src/foam/nanos/controller/MementoUtils.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.controller',
+  name: 'MementoUtils',
```

Comment:
I think we could have a better name than this. Utils makes me think of a library of static utility methods, not a Mixin. Maybe MementoMixin or Memorable.